firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -242,11 +237,13 @@ uint8_t moveStraight(uint8_t motor[], int endCol, int endRow)
     digitalWrite(motor[STEP_PIN], LOW);
     delay(1); // 1 milliSecond
     digitalWrite(motor[STEP_PIN], HIGH);
-
-    // Updating current position per step
-    *currentMotorPos += eighthStepsPerPulse;
   }
 
+    // Updates current position of relevant motor
+    // Not incremented inside of loop to save runtime, and isn't needed as endstop collision
+    // is only exception to end position result, which will be dealt with by calling alignAxis
+    *currentMotorPos += (eighthStepsPerPulse * numSteps);
```

Comment:
Indentation is off here, need to decrease by two spaces. Also line 243 and 244 are a little confusing can u pls change it to one statement per line, something like:
```c
// Updated position is calculated outside of loop to reduce computation/runtime
// If motor collides with endstop, `alignAxis` is called to fix current position
```